polyfill/intersection-observer-test.html

```diff
@@ -27,6 +27,14 @@
   <!-- Setup -->
   <script>mocha.setup({ui:'bdd'});</script>
 
+  <!-- Force polyfill. -->
+  <script>
+    if (false) {
```

Comment:
Did you mean to keep this here?

Reply:
Yes, but I can remove if you don't want it here. As it is, I thought of this as a documentation for how to quickly test the polyfill on a modern browser.

Reply:
I think it's fine to keep it here (to explain how to test, as you say) but I'd remove the `if (false)` and replace it with a comment that says something along the lines of: "comment out the below script to test the default behavior" (or something like that).

Reply:
Done.